packages/sdk/src/modules/base/index.ts

```diff
@@ -90,17 +100,15 @@ export class Base {
 
   protected async _transactionError(
     method: string,
-    eventData: PeaqEventData[]
-  ): Promise<{ documentation: string[]; name: string } | null> {
+    eventData: PeaqEventData
+  ): Promise<{ documentation: string[]; name: string, section: string } | null> {
```

Comment:
The added 'section' return string contains more error information which helps the user debug their problem.